src/commands/blobs/blobs-set.mjs

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+import { promises as fs } from 'fs'
+import { resolve } from 'path'
+
+import { getStore } from '@netlify/blobs'
+
+import { chalk, error as printError } from '../../utils/command-helpers.mjs'
+import requiresSiteInfo from '../../utils/hooks/requires-site-info.mjs'
+
+/**
+ * The blobs:set command
```

Comment:
[question] Can this be used to set object metadata? Or does the CLI support only setting object values?

Reply:
For now, it only works for setting object values.